Makefile

```diff
@@ -202,7 +202,7 @@ endif
 
 nxOMSAutomationWorker:
 	rm -rf output/staging; \
-	VERSION="1.4"; \
+	VERSION="1.5.0.0"; \
```

Comment:
We will test this thoroughly in INT. @Diastro  checked with Weigie and Varun about whether 4 segment version numbers are supported.